pyproject.toml

```diff
@@ -46,9 +46,11 @@ dependencies = [
     'scikit-learn>=1.0.2',
     'orjson>=3.8.6',
     'mpmath>=1.2.1',
-    'plotly>=5.13.0',
+    'plotly>=5.14.1',
     'matplotlib>=3.5.3',
     'circlify>=0.15.0',
+    "wordcloud>=1.8.2.2; python_version < '3.8'",
```

Comment:
word cloud is supported 3.7 up to 3.11. However the 3.11 supported release whilst on github hasnt been deployed to pypi. Hence we should allow for its use from 3.7 to 3.10.

Reply:
> word cloud is supported 3.7 up to 3.11. However the 3.11 supported release whilst on github hasnt been deployed to pypi. Hence we should allow for its use from 3.7 to 3.10.

Also change the test of the pytest.skipif accordingly

Reply:
Sure.

Reply:
Based on testing, it seems that it even causes problems with Python 3.8 on Windows. Also, if you look at their Github actions, you'll see that their tests don't pass for Python 3.8 and above.

Let's be more conservative for this. I will revert to previou commits.